rules/CodeQuality/Rector/LogicalAnd/LogicalToBooleanRector.php

```diff
@@ -55,10 +55,21 @@ public function getNodeTypes(): array
      */
     public function refactor(Node $node): ?Node
     {
-        if ($node instanceof LogicalOr) {
-            return new BooleanOr($node->left, $node->right);
+        $left = $node->left;
+        if ($left instanceof LogicalOr || $left instanceof LogicalAnd) {
+            /** @var BooleanAnd|BooleanOr $left */
+            $left = $this->refactor($left);
+        }
+
+        $right = $node->right;
+        if ($right instanceof LogicalOr || $right instanceof LogicalAnd) {
+            /** @var BooleanAnd|BooleanOr $right */
+            $right = $this->refactor($right);
         }
 
-        return new BooleanAnd($node->left, $node->right);
+        if ($node instanceof LogicalOr) {
+            return new BooleanOr($left, $right);
+        }
```

Comment:
this should has new line after if end, please rebase, then run:

```
composer fix-cs -- rules/CodeQuality/Rector/LogicalAnd/LogicalToBooleanRector.php
bin/rector process rules/CodeQuality/Rector/LogicalAnd/LogicalToBooleanRector.php
```

Reply:
Done